### ISSUE TITLE
Add Ruby 3 support

### ISSUE THREAD

**Issue by ilvez** [phrase/phraseapp-in-context-editor-ruby#48]
Using Ruby 3 specs fail:
```
  1) PhraseApp::InContextEditor::BackendService#translate phrase is disabled different arguments given default array given 
     Failure/Error: I18n.translate_without_phraseapp(*args)
     
     ArgumentError:
       wrong number of arguments (given 2, expected 0..1)
     # /home/ilvez/.rvm/gems/ruby-3.0.0/gems/i18n-1.8.8/lib/i18n.rb:196:in `translate'
     # ./lib/phraseapp-in-context-editor-ruby/backend_service.rb:17:in `translate'
     # ./spec/phraseapp-in-context-editor-ruby/backend_service_spec.rb:27:in `block (3 levels) in <top (required)>'
     # ./spec/phraseapp-in-context-editor-ruby/backend_service_spec.rb:122:in `block (6 levels) in <top (required)>'
```

**Comment by theSoenke** [comment]
@ilvez Thanks for reporting it. We have it on our list and will tackle this.

**Comment by sbruhns** [comment]
fixed with #49
